src/encoded/schemas/cell_annotation.json

```diff
@@ -34,19 +34,27 @@
             "description": "",
             "type":  "array",
             "items": {
-                "type": "string",
-                "linkTo": "Gene"
+                "type": "string"
```

Comment:
The change from object to string requires some additional changes
Both of these properties are embedded in cell_annotation.py - those should be removed
The facets & columns in this schema need to have the '.symbol' removed